### PR TITLE
docs(samples): add query external bigtable using permanent table

### PR DESCRIPTION
**Pull request by pmakani**

### CODE REVIEW

**Comment by stephaniewang526** [comment]
Hi @billyjacobson -- PTAL as well! Thank you.

**Review by stephaniewang526 — APPROVED** [review]

File: samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    String bigtableName = "MY_BIGTABLE_NAME";
+    String datasetName = "MY_DATASET_NAME";
+    String tableName = "MY_TABLE_NAME";
+    String sourceUri =
+        "https://googleapis.com/bigtable/projects/"
+            + projectId
+            + "/instances/"
+            + instanceId
+            + "/tables/"
+            + bigtableName;
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java:50]: Ok, but we usually prefer to see `String.format()`

**Review by lesv — APPROVED** [review]
LGTM for java-samples

File: samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  }
+
+  public static void queryExternalBigtablePerm(
+      String datasetName, String tableName, String sourceUri, String query) {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      BigtableColumn name =
```
Comment (billyjacobson) [inline samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java:62]: same comments from #763 regarding example bigtable schema

Comment (pmakani) [reply samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java:59]: https://github.com/googleapis/java-bigquery/pull/763#discussion_r498215234

File: samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+// Sample to queries an external bigtable data source using a permanent table
+public class QueryExternalBigtablePerm {
+
+  public static void main(String[] args) {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "MY_PROJECT_ID";
+    String instanceId = "MY_INSTANCE_ID";
+    String bigtableName = "MY_BIGTABLE_NAME";
+    String datasetName = "MY_DATASET_NAME";
```
[the remark this answers is not in the document]
Comment (pmakani) [reply samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java]: Done!

File: samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java (hunk 1 above)
Comment (pmakani) [reply samples/snippets/src/main/java/com/example/bigquery/QueryExternalBigtablePerm.java]: Done!

**Comment by pmakani** [comment]
@stephaniewang526 PTAL.

**Review by stephaniewang526 — APPROVED** [review]

**Pull request merged**